tests/basic_correctness/test_prefix_caching.py

```diff
@@ -0,0 +1,62 @@
+"""Compare the outputs of HF and vLLM when using greedy sampling.
+
+It tests prefix caching. Chunked prefill can be enabled by
+enable_prefix_caching=True.
+
+Run `pytest tests/basic_correctness/test_prefix_caching.py`.
```

Comment:
We already have `test_prefix_caching.py`. Can we just add tests to that file?

Reply:
done